clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/github/GitHubArtifactConfiguration.java

```diff
@@ -56,6 +57,7 @@ GitHubArtifactProviderProperties githubArtifactProviderProperties() {
   OkHttpClient okHttpClient;
 
   @Autowired
+  @Lazy
```

Comment:
Hmm, I've since removed the `ObjectMapper` from the other class, this was used to avoid a "bean is already under construction" error. Will try building and running without this now.

Reply:
I learned that bean names are global (thanks @lwander), and the problem wasn't `ObjectMapper` anymore but `OkHttpClient`. Fixed.